trade_remedies_caseworker/templates/organisations/invite_party_complete.html

```diff
@@ -0,0 +1,47 @@
+{% extends "v2/govuk/base_with_form.html" %}
+{% load static %}
+<!-- {    % load text_element %   }
+{% load custom_filters %}
+{% load application_heading %}
+{% load format_date %} -->
+
+{% block back_button %}
```

Comment:
you can just leave this block empty but defined to clear the back button. In that case, Django will overwrite whatever is in the parent back_button block with what you define here, but here it's just empty, so the back button never appears.

```suggestion
{% block back_button %}{% endblock back_button %}
```